apps/twap/src/app/utils/getConditionalOrderId.ts

```diff
@@ -66,11 +50,11 @@ export function twapOrderToStruct(order: TWAPOrder): TWAPOrderStruct {
 
 export function buildTwapOrderParamsStruct(
   chainId: SupportedChainId,
-  twapOrderData: TWAPOrderStruct
+  twapOrderData: OrderStruct
 ): ConditionalOrderParams {
   return {
     handler: TWAP_HANDLER_ADDRESS[chainId],
-    salt: '0x00000000000000000000000000000000000000000000000000000018920d8ce7', // hexZeroPad(Buffer.from(Date.now().toString(16), 'hex'), 32),
+    salt: '0x00000000000000000000000000000000000000000000000000000018920d8ce7',
```

Comment:
I should've added a comment on this as well. In essence, this salt must match between FE and BE to produce the same id, so we need to sync it.

If we use time here, because time on server vs local differ, we end up with mismatching ids.

cc/ @shoom3301